MetaMask/metamask-extension
# Update secret recovery phrase screen

Fixes: [#13229](https://github.com/MetaMask/metamask-extension/issues/13229)

Explanation:  
UX and UI improvements for the Reveal Secret Recovery Phrase page
Figama: https://www.figma.com/file/8z4Csc3wlOIkUz2OP1eTZW/Secure-UX-Improvements?node-id=44%3A1574


Manual testing steps:  
  - Go to Settings -> Reveal Secret Recovery Phrase
  - Check a new design from the Figma for the Secret Recovery Phrase page
  - Enter the password and check Revealed SRP page design
  - Click on another tab in the browser. You should be redirected to the default page. 
  - Copy SRP in the clipboard. It should be deleted after 1 min
